Lib/test/test_inspect.py

```diff
@@ -990,6 +990,11 @@ def f(self):
             with DirsOnSysPath(tempdir):
                 import inspect_actual
                 self.assertIn("correct", inspect.getsource(inspect_actual.A))
+                # --forever will run this test multiple times, each time the test
+                # executes, we create a different temporary directory, and the old
+                # one is removed. So we need to remove the module from sys.modules
+                # to force it to be reloaded.
+                sys.modules.pop("inspect_actual")
```

Comment:
I think the reason for doing this is pretty self-evident, and doesn't really require a comment. If we do have a comment, I think it can be shorter and not mention `--forever` specifically (since there are multiple ways a test can run multiple times in same process, including `-R` also).

Reply:
I shortened it.